### PR TITLE
[APM2-714] Move OCBC Pay Anyone to be at first-level payment option

### PR DESCRIPTION
**Pull request by FhanOmise**
#### 1. Objective

Move `OCBC Pay Anyone` option out from `Mobile Banking` to first-level payment option.

**Related information**:
Task: https://omise.atlassian.net/browse/APM2-714

#### 2. Description of change

- add new offsite payment for ocbc pao
- remove ocbc pao from mobile banking
- set mobile banking display only when checkout is THB

#### 3. Quality assurance

OCBC PAO can make a charge with type OCBC Pay Anyone successfully

**🔧 Environments:**

Specify the details of your test environments, including, for each, the platform version (on which the plugin was run), the Omise plugin version, and the versions of your system software such as PHP or Ruby.

i.e.
- **WooCommerce**: v.6.2.2
- **WordPress**: v5.9.3
- **PHP version**: 7.1.1
- **Omise plugin version**: Omise-WooCommerce 4.19 (optional, in case of submitting a new issue)

**✏️ Details:**

Explain how to manually test this feature.

Test charge creation for OCBC PAO.

#### 4. Impact of the change

OCBC Pay Anyone show in Checkout page.
Screen Shot 2565-05-06 at 09 20 25

#### 5. Priority of change

Normal.

#### 6. Additional Notes

Any further information that you would like to add.

### CODE REVIEW

File: includes/classes/class-omise-image.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		 * @param string $file Image file name with extension such as image.jpg
+		 * @param string $alternate_text Alternate text for the image
+		 * @return string HTML &lt;img&gt; element
+		 */
+		public function get_image( $file, $alternate_text ) {
+			$url = WC_HTTPS::force_https_url( plugin_dir_url( '' ) . 'omise/assets/images/' );
+			return "<img src='$url/$file' class='Omise-Image' style='width: 30px; max-height: 30px;' alt='$alternate_text' />";
+		}
+	}
+}
```
Comment (new4762) [inline includes/classes/class-omise-image.php:18]: EOF

**Review by new4762 — APPROVED** [review]

File: includes/classes/class-omise-image.php (hunk 1 above)
Comment (FhanOmise) [reply includes/classes/class-omise-image.php]: Fixed [37e55ab](https://github.com/omise/omise-woocommerce/pull/265/commits/37e55abf5f514f0c3e913c0c293a335541e775a4)

**Review by tanawin-opn — COMMENTED** [review]
Tested and working correctly krub.

**Review by som-m — APPROVED** [review]
👍

image

image

image

**Review by hpariwat — APPROVED** [review]

**Review by tookaa2 — APPROVED** [review]

**Pull request merged**